doc/source/tutorial/formats.rst

```diff
@@ -31,9 +31,9 @@ specifies all entry types in a given dataset::
     11   Woldemort   Владимир      vladimir    Russian      6
     12   Woldemort   Володимир     volodimir   Ukrainian    6
     9    Harry       Harald        haralt      German       7
-    10   Harry       Harry         hæri        English      7
-    11   Harry       Гарри         gari        Russian      7
-    12   Harry       Гаррi         hari        Ukrainian    7
+    10   Harry       Harry         hæri        English      8
```

Comment:
I'd prefer to leave "harald" cognate with "harry", as etymologically it is the same, right? and it could also be used to later indicate that there is some partial cognacy.